servers/rendering/renderer_rd/shaders/forward_clustered/scene_forward_clustered.glsl

```diff
@@ -2257,42 +2224,94 @@ void fragment_shader(in SceneData scene_data) {
+		// Compute solid bits
 
-		imageStore(albedo_volume_grid, grid_pos, uvec4(albedo16));
+		// Compute normal bits.
+		// Lower 6 are inclusive (depending on axis vector.
 
-		uint facing_bits = 0;
-		const vec3 aniso_dir[6] = vec3[](
-				vec3(1, 0, 0),
-				vec3(0, 1, 0),
-				vec3(0, 0, 1),
-				vec3(-1, 0, 0),
-				vec3(0, -1, 0),
-				vec3(0, 0, -1));
+		// upper 26 are exclusive (With some margin), used to save something closer to the normal.
 
-		vec3 cam_normal = mat3(scene_data.inv_view_matrix) * normalize(normal_interp);
+		const int facing_direction_count = 26;
+		const vec3 facing_directions[26] = vec3[](vec3(-1.0, 0.0, 0.0), vec3(1.0, 0.0, 0.0), vec3(0.0, -1.0, 0.0), vec3(0.0, 1.0, 0.0), vec3(0.0, 0.0, -1.0), vec3(0.0, 0.0, 1.0), vec3(-0.5773502691896258, -0.5773502691896258, -0.5773502691896258), vec3(-0.7071067811865475, -0.7071067811865475, 0.0), vec3(-0.5773502691896258, -0.5773502691896258, 0.5773502691896258), vec3(-0.7071067811865475, 0.0, -0.7071067811865475), vec3(-0.7071067811865475, 0.0, 0.7071067811865475), vec3(-0.5773502691896258, 0.5773502691896258, -0.5773502691896258), vec3(-0.7071067811865475, 0.7071067811865475, 0.0), vec3(-0.5773502691896258, 0.5773502691896258, 0.5773502691896258), vec3(0.0, -0.7071067811865475, -0.7071067811865475), vec3(0.0, -0.7071067811865475, 0.7071067811865475), vec3(0.0, 0.7071067811865475, -0.7071067811865475), vec3(0.0, 0.7071067811865475, 0.7071067811865475), vec3(0.5773502691896258, -0.5773502691896258, -0.5773502691896258), vec3(0.7071067811865475, -0.7071067811865475, 0.0), vec3(0.5773502691896258, -0.5773502691896258, 0.5773502691896258), vec3(0.7071067811865475, 0.0, -0.7071067811865475), vec3(0.7071067811865475, 0.0, 0.7071067811865475), vec3(0.5773502691896258, 0.5773502691896258, -0.5773502691896258), vec3(0.7071067811865475, 0.7071067811865475, 0.0), vec3(0.5773502691896258, 0.5773502691896258, 0.5773502691896258));
 
-		float closest_dist = -1e20;
+		uint bit_normal = 0;
 
-		for (uint i = 0; i < 6; i++) {
-			float d = dot(cam_normal, aniso_dir[i]);
-			if (d > closest_dist) {
-				closest_dist = d;
-				facing_bits = (1 << i);
+		//const float exclusive_threshold = 0.86; // given min cos is 0.70710676908493
+		const float exclusive_threshold = 0.7; // given min cos is 0.70710676908493
+		const float inclusive_threshold = 0.001;
+
+		for (int i = 0; i < facing_direction_count; i++) {
+			float dp = dot(cam_geom_normal, facing_directions[i]);
+
+			if (i < 6 && dp > inclusive_threshold) {
+				bit_normal |= uint(1 << i);
+			}
+
+			if (dp > exclusive_threshold) {
+				bit_normal |= uint(1 << (i + 6));
 			}
 		}
 
+		// Subgroup merge and store normal bits
+
+		{
 #ifdef MOLTENVK_USED
-		imageStore(geom_facing_grid, grid_pos, uvec4(imageLoad(geom_facing_grid, grid_pos).r | facing_bits)); //store facing bits
+			imageStore(geom_normal_bits, igrid_pos, uvec4(imageLoad(geom_normal_bits, igrid_pos).r | (1 << bit_ofs))); //store solid bits
```

Comment:
Should this be
```suggestion
			imageStore(geom_normal_bits, igrid_pos, uvec4(imageLoad(geom_normal_bits, igrid_pos).r | bit_normal)); //store solid bits
```
It seemed to be the one issue with rebasing to master and getting this running on macos